site/docs/guides/configuring-fusuma.md

```diff
@@ -68,6 +69,12 @@ code:
 
 See these options for detail. [babel-plugin-prismjs#configuring-the-plugin](https://github.com/mAAdhaTTah/babel-plugin-prismjs#configuring-the-plugin), [Prism.js Playground](https://prismjs.com/test.html#language=markup)
 
+## MathJax
```

Comment:
Because `remark-html-katex` is used, it should be `KaTeX` rather than `MathJax`.
> Note: KaTeX and MathJax are both LaTeX (syntax) renderer for web browsers.

Reply:
Here is a blog post discussing two libraries:
https://cpplover.blogspot.com/2017/04/katex.html

Reply:
And because MathJax and KaTeX are just a name of the libraries, we might want to introduce this feature as just __Math__ or __LaTeX syntax__ like [Typora's document](https://support.typora.io/Math/) does.

Reply:
Makes sense. So should we change to `KaTex`? And we might consider changing the name of `math` in `.fusumarc`.  /cc @uetchy

Reply:
> So should we change to `KaTex`?
We rather have to change it to `Math` (or `Math (LaTeX)`, as you like). It's plainer than the random library name.
As galeone said, it's ok to leave `math` in `.fusumarc` intact.

Reply:
`Math (KaTeX)`? I prefer `Math (KaTeX)`. Or we should switch from LaTex to MathJax? Sorry but I don't use this feature, so I need to know the best solution. I don't know LaTex is the best or not.😂

Reply:
It's a little confusing over their names...😭

- LaTeX is a __syntax__
- KaTeX is a __library__ that parses LaTeX
- MathJax is also a __library__ that parses LaTeX

So yes, `Math (KaTeX)` is alright. `Math (LaTeX)` is also fine.

MathJax is the most popular library and supporting plenty of syntax from LaTeX. KaTeX is a newcomer which is reportedly __fast__, and __less functional__ than MathJax. For instance, KaTeX does not have support for `\begin{equation}` or `\begin{eqnarray}` hence there is no way to express the following equation in KaTeX.

![](https://math.now.sh?from=%5Cbegin%7Beqnarray%7D%0A1%20%2B%20%5Cfrac%7B%201%20%7D%7B%201%20%2B%20%5Cfrac%7B%201%20%7D%7B%201%20%2B%20%5Cfrac%7B%201%20%7D%7B%201%20%2B%20%5Cddots%20%7D%20%7D%20%7D%0A%3D%20%5Cfrac%7B%201%20%7D%7B%202%20%7D%20%5Cleft(%201%20%2B%20%5Csqrt%7B%205%20%7D%20%5Cright)%0A%5Cend%7Beqnarray%7D)

```latex
\begin{eqnarray}
1 + \frac{ 1 }{ 1 + \frac{ 1 }{ 1 + \frac{ 1 }{ 1 + \ddots } } }
= \frac{ 1 }{ 2 } \left( 1 + \sqrt{ 5 } \right)
\end{eqnarray}
```

Seeing the bright side there is already a remark plugin for KaTeX so choosing it is probably the easiest way to support math syntax.

Reply:
Having full power with MatJax is always sweet of course but it might take more effort to put into action.

Reply:
Thank you for the lovely explanation :bow:

> KaTeX does not have support for \begin{equation} and \begin{eqnarray}

Currently, we use [remark-math](https://github.com/Rokt33r/remark-math) and this library uses `remark-html-katex`, so we should replace `remark-katex` with [`remark-latex`](https://www.npmjs.com/package/remark-latex) if possible?